todo.js

```diff
@@ -28,7 +28,14 @@ function renderTodoList() {
   todos.forEach((todo) => {
     const newTodoItem = document.createElement("li");
 
-    if (todo.editState) {
+    if (todo.doneState) {
+      const doneText = document.createElement("s");
```

Comment:
the `s` html element has accessibility concern. Please have a look [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/s#accessibility_concerns)

Consider using the previous tag for the text but let's add a custom class to it so that when the user makes the todo done, we'll add the class that will apply `text-decoration: line-through`